opencti-worker/src/message_queue_consumer.py

```diff
@@ -13,9 +14,10 @@ class MessageQueueConsumer:  # pylint: disable=too-many-instance-attributes
     consumer_type: Literal["listen", "push"]
     queue_name: str
     pika_parameters: pika.ConnectionParameters
-    execution_pool: ThreadPoolExecutor
+    execution_pool: ThreadPoolExecutor | ThreadPoolSelector
```

Comment:
execution_pool and is_realtime could be abstracted, and provide a single 'submitTask' arg